Finjector.Web/ClientApp/src/components/Details/PpmDetails.tsx

```diff
@@ -4,9 +4,21 @@ import { Approver } from "../../types";
 
 interface PpmDetailsProps {
   ppmProjectManager: Approver;
+  projectCompletionDate: string;
+  projectStartDate: string;
+  projectStatus: string;
+  projectTypeName: string;
+  ppmGlString: string;
```

Comment:
i would just pass the entire ppmDetails as a prop instead of writing each out like this